docs/USING_ADVANCED.md

```diff
@@ -25,7 +25,6 @@ marked.setOptions({
   },
   pedantic: false,
   gfm: true,
-  tables: true,
```

Comment:
Can `tables: true` be used when `gfm: false`? It seems like the answer is no, but I want to make sure there are no other cases I'm missing.

Reply:
no `tables` is like `breaks` and only works when `gfm: true` is set